views/roles.py

```diff
@@ -64,7 +64,7 @@ def role_uri():
             else:
                 # Check for required arguments
                 if name and not query.does_role_name_exist(name):
-                    role = Role(name=name, 
+                    role = Role(None, name=name, 
```

Comment:
The `None` should be removed from the constructor since the constructors for the models were changed in a previously closed pull request.